### DISCUSSION TITLE
Swap out `micromustache` for "full" `mustache.js` (or equivalent)

### DISCUSSION THREAD

**Discussion by gubarez** [directus/directus#15316] [Feature Requests]
I see the mustache render is micromustache, which however doesn't support arrays. Do you have any suggestions, apart from the Run Script operation, as it would be quite unhandy to make it this way. Perhaps we could use mustache.js or handlebars.js?

**Comment by rijkvanzanten** [comment]
There's no other way to template out data in there. I've converted this to a feature request to allow for "full" mustache.js instead of just the `micromustache` rendering 👍🏻

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
